packages/core/src/storages/request_provider.ts

```diff
@@ -445,6 +445,13 @@ export abstract class RequestProvider implements IStorage {
             this._maybeAddRequestToQueueHead(request.id, forefront);
         }, STORAGE_CONSISTENCY_DELAY_MILLIS);
 
+        // Finally, try to delete the request lock (even if the request is not using locks)
+        try {
+            await this.client.deleteRequestLock(request.id, { forefront });
+        } catch {
+            // Oh well
```

Comment:
not sure if this fails for the requests without locks, if not, i would log the error to debug level

Reply:
I think it does fail for requests without the lock, and we usually don't _care_ if we cannot remove a lock. Either some other client has the lock (which shouldn't happen, but if it does, its not useful to log), or the request isn't locked (in which case rip). Only other error is auth but that's by then already guaranteed by the flow of the queue

Reply:
If the lock is not there or if another client locks it, the API will not fail. It is the same pattern as you delete some already deleted Actor.

Reply:
Ok, then it makes sense to log the error as it should generally not be happening unless its something broken, right?

Debug level seems enough I guess, so we don't spam?

Reply:
Agree errors should happen only if internal error happens.

Reply:
Oh I see, just tested the endpoint and it always returns 204 even if it fails

*I don't think thats right at all but 🤷*

Will adjust to log the error then as a debug msg